src/engine/qcommon/qcommon.h

```diff
@@ -285,7 +285,7 @@ You or the server may be running older versions of the game."
 
 #define PROTOCOL_VERSION       86
 
-#define URI_SCHEME             GAMENAME_STRING "://"
+#define URI_SCHEME             va("%s://", current_game->gamename_string)
```

Comment:
Servers would use different port numbers for different instances, and the game logic should be discovered during connection, so I don't think we need a path component to the URI.